packages/framework/ini-files/config-specs.ini

```diff
@@ -2848,13 +2849,15 @@ use PACKAGE-ENABLES|NO-PACKAGE-ENABLES
 
 use COMMON_SPACK_TPLS
 
-opt-set-cmake-var MPI_EXEC_PRE_NUMPROCS_FLAGS                            STRING       : --bind-to;none
```

Comment:
I'd like us to stop aligning columns like this for exactly this reason.  It generates a large diff when you really changed a couple of things.  Don't worry about it for this one, but for the future I do not think keeping the columns aligned is a good idea (https://stackoverflow.com/questions/101958/code-formatting-is-lining-up-similar-lines-ok agrees with me, though that's not exactly an iron-clad source).